API change in SoapySDR-0.8.1. Upstream currently has 3 years of unreleased changes,
including a similar fix, but it was too convoluted to easily import directly.

--- a/src/convenience/convenience.c
+++ b/src/convenience/convenience.c
@@ -488,7 +488,8 @@
 
 	show_device_info(dev);
 
-	if (SoapySDRDevice_setupStream(dev, streamOut, SOAPY_SDR_RX, format, NULL, 0, &stream_args) != 0) {
+	*streamOut = SoapySDRDevice_setupStream(dev, SOAPY_SDR_RX, format, NULL, 0, &stream_args);
+	if (*streamOut == NULL) {
 		fprintf(stderr, "SoapySDRDevice_setupStream failed\n");
 		return -3;
 	}
